src/state/transactions/reducer.ts

```diff
@@ -1,17 +1,179 @@
-import { createReducer } from '@reduxjs/toolkit'
+import { createSlice } from '@reduxjs/toolkit'
+import { TradeType } from '@uniswap/sdk-core'
 
 import { updateVersion } from '../global/actions'
-import {
-  addTransaction,
-  checkedTransaction,
-  clearAllTransactions,
-  finalizeTransaction,
-  SerializableTransactionReceipt,
-  TransactionInfo,
-} from './actions'
+import { VoteOption } from '../governance/types'
 
 const now = () => new Date().getTime()
 
+interface SerializableTransactionReceipt {
```

Comment:
i think we'd typically put these at `transactions/types.ts` for readability

Reply:
Done.